_posts/2023-02-16-announcing-ballerina-2201.4.0-swan-lake-update-4.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog-post
+title: Announcing Ballerina 2201.4.0 (Swan Lake Update 4)
+author: Ballerina Team
+published-date: 16 February 2023
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2023-02-16-announcing-ballerina-2201.4.0-swan-lake-update-4/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are happy to announce a new [Ballerina 2201.4.0 (Swan Lake Update 4)](https://ballerina.io/downloads/) release, which is the fourth update release of Ballerina Swan Lake.
+
+The main highlight of this release is to support creating multiple output bindings when using Azure Functions. From this release onwards, you can define a tuple as a return type and annotate each member of the tuple using an output binding. This requirement is facilitated from the compiler level by the introduction of the ability for tuple members to be annotated via `field` attachment point declarations.
```

Comment:
Let's move this to a separate section. I guess we don't have a key highlight in this release

Reply:
Will be addressed via https://github.com/ballerina-platform/blog/pull/152/commits/e192e27c931f57913c8b99809bc566cb57e1fc88.